tests/hilti/cfg/assign.hlt

```diff
@@ -6,7 +6,22 @@ module foo {
 function uint<64> fn() {
     local auto x = 1;
     x = x + 1;
-    return x;
```

Comment:
was this intentionally removed?

Reply:
Unintentional, but thankfully I didn't hide any bugs.